docs/01-biological-foundations/02-the-central-dogma.md

```diff
@@ -13,7 +13,7 @@ process below.
 The key takeaways for our purposes are as follows:
 
 1.  The complete information for replicating cells is preserved in DNA, which holds the
-    instructions for creating proteins spread across the chromosomes.
+    instructions for creating proteins and is spread across the chromosomes.
```

Comment:
```suggestion
    instructions for creating proteins.
```
This comment merely provides you with a nearly one-click way to remove this clause altogether if you agree that it does not add to the purpose of this sentence.